lib/puppet/vendor/rgen/lib/rgen/metamodel_builder/builder_extensions.rb

```diff
@@ -395,7 +395,7 @@ def get<%= firstToUpper(name) %>
     

```

Comment:
We have some other patches / additions to rgen - see parser_factory.rb. There is also the vendored code loading that could be used to patch the vendored version IIRC.